aixplain/factories/file_factory.py

```diff
@@ -53,14 +60,25 @@ def upload(cls, local_path: Text) -> Text:
             raise FileNotFoundError(f'File Upload Error: local file "{local_path}" not found.')
         # mime type format: {type}/{extension}
         mime_type = filetype.guess_mime(local_path)
-        type_to_max_size = { "audio": MB_50, "application": MB_25, "video": MB_300, "image": MB_25, "other": MB_50 }
+        if mime_type is None:
+            content_type = "text/csv"
+        else:
+            content_type = mime_type
+
+        type_to_max_size = {"audio": MB_50, "application": MB_25, "video": MB_300, "image": MB_25, "other": MB_50}
         if mime_type is None or mime_type.split("/")[0] not in type_to_max_size:
             ftype = "other"
         else:
             ftype = mime_type.split("/")[0]
         if os.path.getsize(local_path) > type_to_max_size[ftype]:
-            raise Exception(f'File Upload Error: local file "{local_path}" of type "{mime_type}" exceeds {type_to_max_size[ftype] / MB_1} MB.')
-        s3_path = upload_data(file_name=local_path)
+            raise Exception(
+                f'File Upload Error: local file "{local_path}" of type "{mime_type}" exceeds {type_to_max_size[ftype] / MB_1} MB.'
+            )
+
+        if is_temp is False:
+            s3_path = upload_data(file_name=os.path.basename(local_path), tags=tags, license=license, is_temp=is_temp, content_type=content_type)
```

Comment:
If the user gives a relative path (e.g. `folder/file.txt`), the document will not be found, since we are using just the base name (`file.txt`). So this is a bug and may be fixed.